src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,183 @@
+import p5 from 'p5' // we need the p5.Color type to declare top-level colors
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# Entries (p5 Template)
+
+[<img
+  src="../../assets/img/Entries/1729.png"
+  width=700
+  style="margin-left: 1em; margin-right: 0.5em"
+/>](../../assets/img/Entries/1729.png)
+
+This visualizer allows you to step back and forth through a sequence, showing
+each entry as a written number.
+
+_This visualizer is meant to be used as a template for new visualizers based on
+the p5.js library. It includes explanatory comments and minimal examples of
+required and commonly used features._
+**/
+
+class P5VisualizerTemplate extends P5Visualizer {
+    // === visualizer name ===
+    // appears in the visualizer list and bundle card titles
+
+    static visualizationName = 'Entries (p5 Template)'
+
+    // === parameters ===
+    // top-level properties that the user can choose while creating the
+    // visualizer bundle
+
+    stepSize = 1 // the default value of this parameter is zero
+
+    /** md
+## Parameters
+    **/
+    params = {
+        /** md
+- **Step size:** How far to step when the user presses an arrow key.
+         **/
```

Comment:
Consider changing to "A positive integer specifying how far to ...". I looked through existing visualizer docs. Most don't really express constraints on parameters, whereas a couple do. If it's your sense this would be a good idea if more visualizers did it, then good to model that in the template.

Reply:
I like this idea. What do you think of the format I've proposed in the Entries and Differences documentation?

Reply:
Seems like a reasonable option to me, generally speaking. Kate?
Do we need italics? At some point, over-fancying docs makes them harder, not easier, to read.
Also, I like using the exact legend from the UI as the heading on the doc page; should we reinforce that by also using that exact text when cross-referencing the parameter? I.e., in the Differences doc for number of rows, say "Positive integer, no larger than entries in top row." or "Positive integer, no larger than Entries in top row." or "Positive integer, no larger than 'Entries in top row.'" ?

Reply:
Left the formatting; we can have Kate eyeball it sometime. Changed the parameter cross reference to use the same label. Resolving.